### DISCUSSION TITLE
CPM.cmake is very slow while generating Visual Studio Project File

### DISCUSSION THREAD

**Discussion by henrya2** [cpm-cmake/CPM.cmake#514] [General]
It is really very slow. And every time, generate Visual Studio Project File even the dependencies have been already up to date and no change.

**Comment by OlivierLDff** [comment]
Are you using `CPM_SOURCE_CACHE`?
You can profile your cmake build to know what is slow: https://www.jetbrains.com/help/clion/cmake-profiling.html#:~:text=With%20CMake%20profiling%2C%20you%20can,in%20Google's%20Trace%20Event%20format.
